### PR TITLE
feat(Automated the data in Logwork and added non-editable userID)

### PR DESCRIPTION
**Pull request by murlidharpathakprogrammer**
Signed-off-by: Murlidhar Pathak <murlidharpathakprogrammer>

### Functionality:
Automated status and work completion input field in logwork.

### Solution:
Automated input fields

### Risk level:
- [ ] high 
- [ ] medium
- [X] low

### How to test:
firebase emulators:start
ng serve --open

Screenshot from 2021-08-24 02-08-33

### CODE REVIEW

**Comment by murlidharpathakprogrammer** [comment]
Automated data and added non-editable userid in logwork

**Review by arjund-06 — APPROVED** [review]
Ship it

File: src/app/body/task-details/log-work/log-work.component.html [hunk 1]
```diff
@@ -17,14 +17,22 @@ <h5 class="active nav-link head" id="exampleModalLabel">Log Work</h5>
             </nav>
             <div class="container modal-body align-items-center px-4" id="logWorkPage">
                 <form *ngIf="!showClose" (ngSubmit)="submit()" #form="ngForm">
+                    <div class="row px pt-2">
+                        <div class="col-md"></div>
+                        <div class="col-md-5 pl-2">
+                            <span>{{task.Id}}</span>
+                        </div>
```
Comment (viv9k) [inline src/app/body/task-details/log-work/log-work.component.html:24]: Add a label to this task id.

**Review by viv9k — CHANGES_REQUESTED** [review]
Please do the required changes and submit asap.

File: src/app/body/task-details/log-work/log-work.component.html [hunk 2]
```diff
@@ -17,14 +17,23 @@ <h5 class="active nav-link head" id="exampleModalLabel">Log Work</h5>
             </nav>
             <div class="container modal-body align-items-center px-4" id="logWorkPage">
                 <form *ngIf="!showClose" (ngSubmit)="submit()" #form="ngForm">
+                    <div class="row px pt-2">
```
Comment (viv9k) [inline src/app/body/task-details/log-work/log-work.component.html:20]: What is px doing here

**Review by viv9k — CHANGES_REQUESTED** [review]
Please do the required change and also show demo in today evening meeting

File: src/app/body/task-details/log-work/log-work.component.html (hunk 2 above)
Comment (murlidharpathakprogrammer) [reply src/app/body/task-details/log-work/log-work.component.html:20]: Actually that was part of the previous code written by someone else.

Comment (murlidharpathakprogrammer) [reply src/app/body/task-details/log-work/log-work.component.html:20]: Actually that was part of the previous code written by someone else.

**Comment by murlidharpathakprogrammer** [comment]
> Please do the required change and also show demo in today evening meeting

done

File: src/app/body/task-details/log-work/log-work.component.html (hunk 2 above)
Comment (viv9k) [reply src/app/body/task-details/log-work/log-work.component.html]: It's showing as your change

**Review by viv9k — APPROVED** [review]
Ship it

**Pull request merged**